### PR TITLE
Update SECURITY.md

### PR DESCRIPTION
**Pull request by loosebazooka**

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -2,10 +2,14 @@
...
-Debian tracking information at https://github.com/GoogleContainerTools/distroless/blob/main/checksums.bzl is updated daily
+Debian tracking information at https://github.com/GoogleContainerTools/distroless/blob/main/debian_archives.bzl and generally is updated within 48 hours of debian publishing an update.
 
 ## Reporting a Vulnerability
 
-If a distroless image you are using contains a CVE or other vulnerability, please let the team know by creating an issue and pointing to the CVE or vulnerability disclosure. Please include a link to the package in question from the debian package index for debian 10/buster (https://packages.debian.org/index)
+If a distroless image you are using contains a CVE or other vulnerability:
+- find the appropriate debian security-tracker notice: `https://security-tracker.debian.org/tracker/CVE-XXXX-YYYYY`, for [example](https://security-tracker.debian.org/tracker/CVE-2022-21476).
+- check if a fix is avialable for the appropriate debian version in the main/security channels.
+  - if a fix is not yet availalb, do not file a bug, track it in your internal tracker until one becomes available.
```
Comment (briandealwis) [inline SECURITY.md:14]:
```suggestion
- check if a fix is available for the appropriate debian version in the main/security channels.
  - if a fix is not yet available, do not file a bug, track it in your internal tracker until one becomes available.
```

File: SECURITY.md [hunk 2]
```diff
@@ -2,10 +2,14 @@
...
-Distroless currently tracks debian 10 ([buster](https://packages.debian.org/buster/)) for packages. Distroless images are updated as debian packages are updated. No updates are created for earlier releases.
+Distroless currently tracks debian 11 ([buster](https://packages.debian.org/bullseye/) current) and debian 10 ([buster](https://packages.debian.org/bullseye/)) packages.
 
-Debian tracking information at https://github.com/GoogleContainerTools/distroless/blob/main/checksums.bzl is updated daily
+Debian tracking information at https://github.com/GoogleContainerTools/distroless/blob/main/debian_archives.bzl and generally is updated within 48 hours of debian publishing an update.
 
 ## Reporting a Vulnerability
 
-If a distroless image you are using contains a CVE or other vulnerability, please let the team know by creating an issue and pointing to the CVE or vulnerability disclosure. Please include a link to the package in question from the debian package index for debian 10/buster (https://packages.debian.org/index)
+If a distroless image you are using contains a CVE or other vulnerability:
```
Comment (briandealwis) [inline SECURITY.md:11]: It would be useful to turn this into a numbered list so that it's clear they are steps.

File: SECURITY.md [hunk 3]
```diff
@@ -2,10 +2,14 @@
...
+- find the appropriate debian security-tracker notice: `https://security-tracker.debian.org/tracker/CVE-XXXX-YYYYY`, for [example](https://security-tracker.debian.org/tracker/CVE-2022-21476).
+- check if a fix is avialable for the appropriate debian version in the main/security channels.
```
Comment (briandealwis) [inline SECURITY.md:13]: Is there a link to these channels?

**Review by briandealwis — APPROVED** [review]
Teeny tiny nits

File: SECURITY.md (hunk 3 above)
Comment (loosebazooka) [reply SECURITY.md]: I guess I could rephrase this a bit. But the debian security tracker usually says what fix is available where.

File: SECURITY.md (hunk 2 above)
Comment (loosebazooka) [reply SECURITY.md:11]: done

**Pull request merged**